src/main/java/picard/sam/markduplicates/util/LibraryIdGenerator.java

```diff
@@ -49,7 +49,9 @@
     private short nextLibraryId = 1;
     private final Map<String, DuplicationMetrics> metricsByLibrary = new HashMap<String, DuplicationMetrics>();
     private final Histogram<Short> opticalDuplicatesByLibraryId = new Histogram<Short>();
-
+    public final Histogram<Double> duplicateCountHist = new Histogram<Double>("set_size", "all_sets");
```

Comment:
type arguments unnecessary here as java infers them from the variable declaration
```suggestion
    public final Histogram<Double> duplicateCountHist = new Histogram<>("set_size", "all_sets");
```